DownFall/Views/BackpackView.swift

```diff
@@ -16,79 +16,226 @@ import SpriteKit
         
-        ctaButton = Button(size: CGSize(width: Style.Backpack.ctaButton, height: 125.0), delegate: self, identifier: .backpackUse, precedence: .foreground, fontSize: UIFont.extraLargeSize, fontColor: UIColor.black, backgroundColor: .highlightGold)
+        //cta button
+        ctaButton = Button(size: CGSize(width: Style.Backpack.ctaButton, height: 100.0), delegate: self, identifier: .backpackUse, precedence: .foreground, fontSize: UIFont.extraLargeSize, fontColor: UIColor.black, backgroundColor: .highlightGold)
         ctaButton?.position = CGPoint.positionThis(ctaButton?.frame, outside: descriptionArea?.frame, anchor: .right, align: .top, padding: -Style.Padding.more)
         
         
-        addOptionalChild(itemArea)
-        addOptionalChild(descriptionArea)
-        addOptionalChild(ctaButton)
+        //cancel button
+        self.cancelButton = Button(size: CGSize(width: Style.Backpack.ctaButton, height: 100.0), delegate: self, identifier: .backpackCancel, precedence: .foreground, fontSize: UIFont.extraLargeSize, fontColor: UIColor.red, backgroundColor: UIColor.eggshellWhite)
+        self.cancelButton?.position = CGPoint.positionThis(self.cancelButton?.frame, outsideOf: ctaButton?.frame, verticality: .bottom, spacing: Style.Padding.more)
+        
+        //next button
+        self.nextItemsButton = Button(size: CGSize(width: Style.Backpack.ctaButton, height: 100.0), delegate: self, identifier: .backpackCancel, precedence: .foreground, fontSize: UIFont.extraLargeSize, fontColor: UIColor.black, backgroundColor: UIColor.menuPurple)
+        self.nextItemsButton?.position = CGPoint.positionThis(self.nextItemsButton?.frame, outsideOf: cancelButton?.frame, verticality: .bottom, spacing: Style.Padding.more)
+        
+        
+        inventoryArea.addOptionalChild(itemArea)
+        inventoryArea.addOptionalChild(descriptionArea)
+        inventoryArea.addOptionalChild(ctaButton)
+        inventoryArea.addOptionalChild(cancelButton)
+        inventoryArea.addOptionalChild(nextItemsButton)
+        
+        updateCTAButton(with: nil)
+        
+//        background.isUserInteractionEnabled = true
+//        inventoryArea.isUserInteractionEnabled = true
+//        targetingArea.isUserInteractionEnabled = true
+        
+        viewContainer.addChild(self.background)
+//        viewContainer.addChild(self.targetingArea)
+        viewContainer.addChild(self.inventoryArea)
+        
+        self.addChild(viewContainer)
+    
+        self.isUserInteractionEnabled = true
+    }
+    
+    
+    func updated() {
+        updateToastMessage()
+        updateReticles()
+        updateDescription(with: viewModel.ability)
+        updateCTAButton(with: viewModel.ability)
+        updateTargetArea()
+    }
+    
+    
+    func updateTargetArea() {
+        if viewModel.ability == nil {
+            targetingArea.removeFromParent()
+        } else {
+            targetingArea.removeFromParent()
```

Comment:
I dont think we need to remove this.  But it might be nice to create a `addChildSafely(child: SKNode)` that checks to see if the child has a parent and if it does, removes it from the parent and then adds it